### ISSUE TITLE
can we recover tests from the same source as fft code?

### ISSUE THREAD

**Issue by edwardhartnett** [NOAA-EMC/NCEPLIBS-sp#51]
@kgerheiser did you tell me that you replaced the FFT code with a third-party fft library?

Does that library have any testing in it's native home?

**Comment by kgerheiser** [comment]
Been a while since I've done anything with this:

https://github.com/NOAA-EMC/NCEPLIBS-sp/tree/feature/new-solver-and-fft

I replaced the fft code we have with `fftw` (a popular open source fft package) and was able to reproduce results. 

http://www.fftw.org

**Comment by aerorahul** [comment]
we do have the build script to build fftw in hpc-stack.

**Comment by edwardhartnett** [comment]
OK, but this is not currently merged with the code?

BTW how did you test that you reproduced results?

@aerorahul are you saying that fftw is used elsewhere in the stack?

**Comment by aerorahul** [comment]
It is not used in any application at EMC. Atleast nothing that involves the global model.
It is used in JEDI applications (although) it is currently optional; i.e. if not found, it disables certain features.  Features that we will want to be turned ON soon enough.

**Comment by kgerheiser** [comment]
It's not merged. It just sort of petered out. It's mostly complete, though.

> BTW how did you test that you reproduced results?

 It wasn't super comprehensive, but I had a simple fft test that ran some trivial data back and forth trough each fft and then I checked that the results matched. Not sure I have them anymore (we didn't have any testing infrastructure) and I think I deleted them locally.  Easy enough to re-create though.
